File: exercicios-kotlin/src/exercicio-1.kt
// Crie um programa que receba um número e diga se ele é par ou ímpar

fun main() {
// Input do numero
    print("Digite um número: ")
    val parOuImpar = readLine()!!.toInt()

// Verificar se o numero é par ou impar utilizando a operação %
    if (parOuImpar % 2 == 0) {
        println("$parOuImpar é um número par.")
    } else {
        println("$parOuImpar é um número ímpar.")
    }
}

File: exerciocios-grupo/exercicio-1.py
# Crie um programa que receba um número e diga se ele é par ou ímpar

# Receber um número do usuário
parOuImpar = int(input("Digite um número: "))

# Verificar se o número é par ou ímpar usando a operação de módulo
if parOuImpar % 2 == 0:
    print(f"{parOuImpar} é um número par.")
else:
    print(f"{parOuImpar} é um número ímpar.")
